src/components/Attachments/AttachmentCarousel/CarouselItem.js

```diff
@@ -108,6 +108,7 @@ function CarouselItem({item, onPress, isFocused, isModalHovered}) {
                     isHovered={isModalHovered}
                     isFocused={isFocused}
                     optionalVideoDuration={item.duration}
+                    isUsedInCarousel
```

Comment:
This caused this a regression here: https://github.com/Expensify/App/issues/37750